compile/error.go

```diff
@@ -64,6 +64,14 @@ func (e includeAsDisabledError) Error() string {
 	return "include-as syntax is currently disabled"
 }
 
+// includingHyphenatedFileNotAllowed is raised when the user attempts to include hyphenated thrift file which is not
```

Comment:
break up the comment line to 80 chars width

Reply:
And re-word please. Adding that it is not allowed is redundant since this is an error type:
```
// includingHyphenatedFileNotAllowed is raised when the user attempts to include hyphenated file names.
```